include/api.php

```diff
@@ -3057,7 +3058,7 @@ function api_oauth_access_token($type){
 
 
 	function api_fr_photos_list($type) {
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.